library/defrag.c

```diff
@@ -159,6 +159,11 @@ int eblob_defrag_in_dir(struct eblob_backend *b, char *chunks_dir)
 	eblob_stat_set(b->stat, EBLOB_GST_DATASORT_VIEW_USED_NUMBER, 0);
 	eblob_stat_set(b->stat, EBLOB_GST_DATASORT_SORTED_VIEW_USED_NUMBER, 0);
 	eblob_stat_set(b->stat, EBLOB_GST_DATASORT_SINGLE_PASS_VIEW_USED_NUMBER, 0);
+	eblob_stat_set(b->stat, EBLOB_GST_DATASORT_NUMBER_BLOBS, 0);
```

Comment:
maybe extract all stat initialization for readability?

Reply:
done